definitions/output/reports/cwv_tech_technologies.js

```diff
@@ -6,19 +6,68 @@ publish('cwv_tech_technologies', {
   tags: ['crux_ready']
 }).query(ctx => `
 /* {"dataform_trigger": "report_cwv_tech_complete", "name": "technologies", "type": "dict"} */
+WITH pages AS (
+  SELECT DISTINCT
+    client,
+    root_page AS origin,
+    tech.technology
+  FROM ${ctx.ref('crawl', 'pages')},
+    UNNEST(technologies) AS tech
+  WHERE
+    date = '${pastMonth}'
+    ${constants.devRankFilter}
+), tech_origins AS (
+  SELECT
+    client,
+    technology,
+    COUNT(origin) AS origins
+  FROM pages
+  GROUP BY
+    client,
+    technology
+), technologies AS (
+  SELECT
+    name AS technology,
+    description,
+    ARRAY_TO_STRING(categories, ', ') AS category,
+    categories AS category_obj,
+    NULL AS similar_technologies
+  FROM ${ctx.ref('wappalyzer', 'apps')}
+), total_pages AS (
+  SELECT
+    client,
+    COUNT(DISTINCT origin) AS origins
+  FROM pages
+  GROUP BY client
+)
+
 SELECT
   client,
-  app AS technology,
+  technology,
   description,
   category,
-  SPLIT(category, ",") AS category_obj,
+  category_obj,
+  similar_technologies,
+  COALESCE(origins, 0) AS origins
+FROM tech_origins
+INNER JOIN technologies
+USING(technology)
+
+UNION ALL
+
+SELECT
+  client,
+  'ALL' AS technology,
+  NULL AS description,
```

Comment:
WDYT worth adding a description?
```suggestion
  'All technologies' AS description,
```

Reply:
I thought it's already taken care of on the frontend:
<img width="1511" alt="Screenshot 2025-01-14 at 20 26 30" src="https://github.com/user-attachments/assets/65fef802-21a1-483a-ab99-0c0a1f3091aa" />

Reply:
Yeah, redundant